### ISSUE TITLE
Error after fab vagrant setup_vagrant

### ISSUE THREAD

**Issue by kkoshijima** [nsh87/shinyVM#4]
Hi, 

Thank you for the thorough instructions and the guidelines.
However, I'm facing a problem with installing shinyVM.

After I did `fab vagrant setup_vagrant`, I got an error with the following message.
I would really appreciate it if you could help me troubleshoot.

Best, 

Kensuke Koshijima

```
$ fab vagrant setup_vagrant

[127.0.0.1:2222] Executing task 'setup_vagrant'
[127.0.0.1:2222] sudo: sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
[127.0.0.1:2222] out: Executing: gpg --ignore-time-conflict --no-options --no-default-keyring --homedir /tmp/tmp.mwaDtpo9b9 --no-auto-check-trustdb --trust-model always --keyring /etc/apt/trusted.gpg --primary-keyring /etc/apt/trusted.gpg --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
[127.0.0.1:2222] out: gpg: requesting key E084DAB9 from hkp server keyserver.ubuntu.com
[127.0.0.1:2222] out: gpg: keyserver timed out
[127.0.0.1:2222] out: gpg: keyserver receive failed: keyserver error
[127.0.0.1:2222] out: 


Fatal error: sudo() received nonzero return code 2 while executing!

Requested: sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
Executed: sudo -S -p 'sudo password:'  /bin/bash -l -c "sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9"

Aborting.
Disconnecting from 127.0.0.1:2222... done.
sudo() received nonzero return code 2 while executing!

Requested: sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9
Executed: sudo -S -p 'sudo password:'  /bin/bash -l -c "sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E084DAB9"
```

**Comment by nsh87** [comment]
Everything appears fine, at least as far as the commands being executed, and the key is there so it shouldn't be timing out. I'm guessing it's an issue with your firewall.

I just pushed a change (4ec05eb558f93d48a0a329b8f5ff054f3c9758e6), pull the commit and try again. It should force the adding of the key through port 80, which should work. See [here](http://unix.stackexchange.com/questions/75892/keyserver-timed-out-when-trying-to-add-a-gpg-public-key) for more info.

**Comment by kkoshijima** [comment]
Whoa!!! Awesome! It worked!!

Thank you very much for your support.

**Comment by nsh87** [comment]
You're welcome.

Fixed by 4ec05eb558f93d48a0a329b8f5ff054f3c9758e6
